### PR TITLE
WIP: Update CLI prefs

### PR DESCRIPTION
**Pull request by joelthe1**

### CODE REVIEW

File: cli.js [hunk 1]
```diff
@@ -1,26 +1,34 @@
 #!/usr/bin/env node
 const fs = require('fs');
 const { argv } = require('yargs')
-  .command('* <file-path>', 'parse and convert the input file', (yargs) => {
-    yargs.positional('file-path', {
-      describe: 'the input USFM or JSON file to be parsed and converted',
+  .command('* <file>', 'Parse/validate USFM 3.x to/from JSON.', (yargs) => {
```
Comment (kavitharaju) [inline cli.js:4]: Here we can give a USFM or JSON file as input. But only parsing operation will be done on it, not validate. If the file is not valid, it throws an error(not True/False, like in validate() method). Does this convey that correctly?

**Review by kavitharaju — COMMENTED** [review]
I see that the main change is the use of --output(and -o) instead of --format. I shall make required changes in tests for that.

**Comment by kavitharaju** [comment]
Made required changes in tests.

**Review by kavitharaju — APPROVED** [review]

File: cli.js (hunk 1 above)
Comment (joelthe1) [reply cli.js:4]: So, the output right now would be:
```
usfm-grammar <file>

Parse/validate USFM 3.x to/from JSON.

Positionals:
  file  The path of the USFM or JSON file to be parsed and/or converted. By
        default, auto-detects input USFM and converts it into JSON and
        vice-versa.

Options:
  -l, --level    Level of strictness in parsing. This defaults to `strict`.
                                                            [choices: "relaxed"]
      --filter   Filter out content from input USFM. Not applicable for input
                 JSON or for CSV/TSV output.              [choices: "scripture"]
  -o, --output   The output format to convert input into.
                                         [choices: "csv", "tsv", "usfm", "json"]
  -h, --help     Show help                                             [boolean]
  -v, --version  Show version number                                   [boolean]
```

Here, the string `Parse/validate USFM 3.x to/from JSON.` is a one line description of the tool itself. This is similar with how you would see the help text using `man` in GNU/Linux. More details can then be found on further reading or via the README.

**Comment by joelthe1** [comment]
The `npm run test` fails for the CLI tests for me. Are there any prerequisites that need to be met before I can have those tests run successfully?

**Comment by kavitharaju** [comment]
Those tests use two additional libraries. 
1. [child_process](https://nodejs.org/api/child_process.html#child_process_child_process). This is used to spawn a new process from the node program. This was causing trouble when run on github actions as well. I haven't set anything on my system to run it, though. 

2. concat-stream. which is a newly installed library.

**Pull request merged**